examples/polypharmacy_gripnet/config.py

```diff
@@ -1,55 +1,57 @@
-from typing import List, Tuple
-
+"""
+Default configurations for Polypharmacy Side Effect Prediction using GripNet
+"""
 from yacs.config import CfgNode
 
 # ---------------------------------------------------------
 # Config definition
 # ---------------------------------------------------------
 
-C = CfgNode()
+_C = CfgNode()
 
 # ---------------------------------------------------------
 # Dataset
 # ---------------------------------------------------------
-C.DATASET = CfgNode()
-C.DATASET.ROOT = "./data"
-C.DATASET.NAME = "pose"
-C.DATASET.URL = "https://github.com/pykale/data/raw/main/graphs/pose_pyg_2.pt"
+_C.DATASET = CfgNode()
+_C.DATASET.ROOT = "./data"
+_C.DATASET.NAME = "pose"
+_C.DATASET.URL = "https://github.com/pykale/data/raw/main/graphs/pose_pyg_2.pt"
 
 # ---------------------------------------------------------
 # Solver
 # ---------------------------------------------------------
-C.SOLVER = CfgNode()
-C.SOLVER.SEED = 1111
-C.SOLVER.BASE_LR = 0.01
-C.SOLVER.EPSILON = 1e-10
-C.SOLVER.MAX_EPOCHS = 66
-C.SOLVER.LOG_EVERY_N_STEPS = 1
+_C.SOLVER = CfgNode()
+_C.SOLVER.SEED = 1111
+_C.SOLVER.BASE_LR = 0.01
+_C.SOLVER.EPSILON = 1e-10
+_C.SOLVER.MAX_EPOCHS = 66
+_C.SOLVER.LOG_EVERY_N_STEPS = 1
 
 # ---------------------------------------------------------
 # GripNet supervertex configs
 # ---------------------------------------------------------
-C.GRIPN_SV1 = CfgNode()
-C.GRIPN_SV1.NAME = "protein"
-C.GRIPN_SV1.INTER_FEAT_CHANNELS = 16
-C.GRIPN_SV1.INTER_AGG_CHANNELS_LIST = [16, 16]
-C.GRIPN_SV1.EXTER_AGG_CHANNELS_LIST = []
+_C.GRIPN_SV1 = CfgNode()
+_C.GRIPN_SV1.NAME = "protein"
+_C.GRIPN_SV1.INTER_FEAT_CHANNELS = 16
+_C.GRIPN_SV1.INTER_AGG_CHANNELS_LIST = [16, 16]
+_C.GRIPN_SV1.EXTER_AGG_CHANNELS_LIST = []
 # Elements in `EXTER_AGG_CHANNELS_LIST` should be a list of combinations of supervertex name (str) and embedding
```

Comment:
@peizhenbai Are these commented lines needed?

Reply:
Yes, the comments are left to explain the function of EXTER_AGG_CHANNELS_LIST. I think we can keep it if there are no other concerns.